test/accessibility/edu-benefits-508.spec.js

```diff
@@ -8,7 +8,7 @@ module.exports = E2eHelpers.createE2eTest(
 
     // Ensure introduction page renders.
     client
-      .url(`${E2eHelpers.baseUrl}/education/apply-for-education-benefits/application/`)
+      .url(`${E2eHelpers.baseUrl}/education/apply-for-education-benefits/application/1990/`)
```

Comment:
I think these changes to 508/e2e tests will break them. Those tests can't open urls that are partially controlled by react-router.